index.js

```diff
@@ -0,0 +1,183 @@
+"use strict";
+
+const path = require('path');
+const runSequence = require('run-sequence');
+const streamqueue = require('streamqueue');
+const remoteSrc = require('gulp-remote-src');
+const concat = require('gulp-concat');
+const URL = require('url');
+const karmaServer = require('karma').Server;
+
+class Erik {
+  /**
+   * @param {Object} options
+   * @param {Object} options.gulp - The gulp instance with which to register Erik's tasks.
+   * @param {Boolean} [options.watch] - When watch is set, Erik will watch for changes to your
+   * bundled spec and re-run the test suite when changes occur.
```

Comment:
I dropped the `combine` task. I thought it was necessary, but I think the errors I was encountering were an artifact of me using incorrect filepaths. Passing the specPath and bundled-deps path directly to Karma works just fine.